### DISCUSSION TITLE
Apple native thunderbolt firmware with Windows

### DISCUSSION THREAD

**Discussion by ivan-ef** [osy86/HaC-Mini#1070] [Q&A] [answered]
Why doesn't thunderbolt with Apple native firmware work on Windows? It does work on real Macs, so what's the difference?

**Comment by osy** [comment, marked as answer]
It’s because we break the signature of the firmware (as intel signed firmware doesn’t work on apple machines and apple signed firmwares doesn’t work on literally any other machine). Luckily when the signature is broken, the parts apple depend on still works but windows doesn’t.
